Added automatic tracking sessions configurations (#436)
opengisch/QField-docs: documentation/how-to/tracking.en.md

*Stop tracking* button.

!![](../assets/images/tracking-stop.png)

> ## Project Tracking Sessions Auto-Start Feature

Author: Much better!

> The feature enables tracking sessions to automatically start upon project load, with the option for users to fill out a feature form unless the 'Hide Form on Add Feature' toggle is enabled for the project's layer, but the default attribute values are used.

Author: Great!

!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)

### Implementation

> 

Reviewer: ```suggestion
:material-monitor: Desktop preparation

```

> For implement it enter to the layer properties and in the QField panel activate "Tracking Session" and specify the your requirement for the tracking.

Reviewer: ```suggestion
The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the qfieldsync plugin is installed in your QGIS profile.
```

*Stop tracking* button.

!![](../assets/images/tracking-stop.png)

## Predefined Project Tracking Sessions

This functionality enables one or more position tracking sessions to automatically start upon project load. Users will be presented with a feature form popup as the tracking session begins to fill in attributes. If the vector layer used to track has the 'Hide Form on Add Feature' mode selected, the feature form will be skipped. 

!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)

### Configure a Project Tracking Session
:material-monitor: Desktop preparation

> The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the qfieldsync plugin is installed in your QGIS profile.

Reviewer 2: On line 75 we use single quotes, here we use double quotes. Is there any kind of convention or something?

Reviewer 2: ```suggestion
The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the QFieldSync plugin is installed in your QGIS profile.
```

Author: I think is better with double quote!

> !![Activating Autimatic Tracking sessions QFieldSync](../assets/images/automatic-tracking-session.png)
> 
> !![Tracking activated automatically in QField](../assets/images/qfield-tracking-session.png,300px)

Reviewer 2: ```suggestion
!![Activating Automatic Tracking sessions QFieldSync](../assets/images/automatic-tracking-session.png)

!![Tracking activated automatically in QField](../assets/images/qfield-tracking-session.png,300px)
```

One is written in Capitals, the other not, any reason? If it is a name, probably quoting can help?

Author: Yes, is better to use quotes
